spring-cloud-gcp-autoconfigure/src/test/java/org/springframework/cloud/gcp/autoconfigure/pubsub/it/PubSubChannelAdaptersIntegrationTests.java

```diff
@@ -324,7 +390,7 @@ public SubscriberFactory defaultSubscriberFactory(
 				GcpProjectIdProvider projectIdProvider,
 				CredentialsProvider credentialsProvider) {
 			if (pubSubAdmin.getSubscription(this.subscriptionName) == null) {
-				pubSubAdmin.createSubscription(this.subscriptionName, this.topicName);
+				pubSubAdmin.createSubscription(this.subscriptionName, this.topicName, 30);
```

Comment:
In the interests of making this test pass faster, can we go lower than 30s?

Reply:
Yes, 10 seconds is the minimum.